cmd/agent/app/integrations.go

```diff
@@ -330,20 +347,35 @@ func pip(args []string) error {
 }
 
 func install(cmd *cobra.Command, args []string) error {
-	if err := validateArgs(args); err != nil {
+	if err := validateArgs(args, localWheel); err != nil {
 		return err
 	}
 
-	// Additional verification for installation
-	if len(strings.Split(args[0], "==")) != 2 {
-		return fmt.Errorf("you must specify a version to install with <package>==<version>")
-	}
-
 	cachePath, err := getTUFPipCachePath()
 	if err != nil {
 		return err
 	}
 
+	if localWheel {
+		// Specific case when installing from locally available wheel
+		// No compatibility verifications are performed, just install the wheel (with --no-deps still)
+		pipArgs := []string{
+			"install",
+			"--cache-dir", cachePath,
```

Comment:
I would add this bit just to be safe:

```
		// We replace the PyPI index with our own by default, in order to prevent
		// accidental installation of Datadog or even third-party packages from
		// PyPI.
		"--index-url", tufIndex,
```

In fact, I would use the same `pipArgs` that comes later as the default no matter whether local installation or not. So this means moving that definition earlier.